### PR TITLE
feat: [SRM-12081]: adding icons for slos listing page

### PR DESCRIPTION
**Pull request by harshilbits**
You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

File: packages/icons/src/grid-white.svg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (abhinavrastogi-harness) [inline packages/icons/src/grid-white.svg:1]: pls follow https://harness.atlassian.net/wiki/spaces/PLATFORM/pages/1963262066/SVG+Guidelines+Best+Practices

**Review by abhinavrastogi-harness — CHANGES_REQUESTED** [review]
please follow the SVG best practices for all icons:
https://harness.atlassian.net/wiki/spaces/PLATFORM/pages/1963262066/SVG+Guidelines+Best+Practices

File: packages/icons/src/grid-white.svg (hunk 1 above)
Comment (harshilbits) [reply packages/icons/src/grid-white.svg]: Done.

**Review by deepesh-ui — APPROVED** [review]

**Review by devinder-harness — APPROVED** [review]

File: packages/icons/src/execution-warning.svg [hunk 2]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg fill="currentColor" viewBox="0 0 19 15" xmlns="http://www.w3.org/2000/svg"><path d="M15.964 15H3.027C.702 15-.737 12.787.392 11.021L6.29 1.339c1.152-1.785 5.261-1.785 6.412 0l5.897 9.682C19.75 12.768 18.288 15 15.964 15z" color="#fff"/><path d="M4.23 13.75h10.408c1.713 0 2.79-1.77 1.941-3.157l-5.22-8.53c-.848-1.417-3.002-1.417-3.85 0l-5.22 8.53c-.832 1.402.229 3.157 1.941 3.157zm5.22-1.894c-.456 0-.832-.354-.832-.785 0-.431.376-.786.832-.786.457 0 .832.354.816.801.016.416-.375.77-.816.77zm-.212-7.083c.392-.093.8.092.963.446.049.123.081.247.081.385-.016.385-.049.785-.081 1.17-.033.601-.082 1.217-.114 1.817-.017.2-.017.37-.017.57a.6.6 0 01-.62.57c-.342 0-.603-.231-.62-.554-.048-.94-.114-1.879-.163-2.818l-.049-.755a.82.82 0 01.62-.831z"/></svg>
```
Comment (abhinavrastogi-harness) [inline packages/icons/src/execution-warning.svg:1]: is this rendering correctly? looks broken in the github preview

File: packages/icons/src/remove-minus.svg [hunk 3]
```diff
@@ -1 +1 @@
-<svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 14 14"><path d="M7 0c3.86 0 7 3.14 7 7s-3.14 7-7 7-7-3.14-7-7 3.14-7 7-7zm0 12.916A5.922 5.922 0 0012.916 7 5.922 5.922 0 007 1.084 5.922 5.922 0 001.084 7 5.922 5.922 0 007 12.916z" fill="#B0B1C4"/><path d="M4.29 6.408h5.422a.542.542 0 010 1.085H4.29a.542.542 0 010-1.085z" fill="#B0B1C4"/></svg>
\ No newline at end of file
+<svg fill="currentColor" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 14 14"><path d="M7 0c3.86 0 7 3.14 7 7s-3.14 7-7 7-7-3.14-7-7 3.14-7 7-7zm0 12.916A5.922 5.922 0 0012.916 7 5.922 5.922 0 007 1.084 5.922 5.922 0 001.084 7 5.922 5.922 0 007 12.916z" fill="#B0B1C4"/><path d="M4.29 6.408h5.422a.542.542 0 010 1.085H4.29a.542.542 0 010-1.085z" fill="#B0B1C4"/></svg>
```
Comment (abhinavrastogi-harness) [inline packages/icons/src/remove-minus.svg:1]: remove hex color from fill

File: packages/icons/src/warning-icon.svg [hunk 4]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg viewBox="0 0 32 32" fill="currentColor"  xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M16 28.988c-7.179 0-13-5.82-13-13s5.821-13 13-13c7.18 0 13 5.82 13 13s-5.82 13-13 13Zm1.501-6.493a1.5 1.5 0 1 0-3 0 1.5 1.5 0 0 0 3 0Zm-1.5-3.517a1 1 0 0 0 1-1V9.026a1 1 0 1 0-2 0v8.952a1 1 0 0 0 1 1Z" fill="#FF7B26"/><path fill-rule="evenodd" clip-rule="evenodd" d="M16 27.988c-6.627 0-12-5.373-12-12 0-6.628 5.373-12 12-12 6.628 0 12 5.372 12 12 0 6.627-5.372 12-12 12Zm2.501-5.493a2.5 2.5 0 1 0-5 0 2.5 2.5 0 0 0 5 0Zm-2.5-2.517a2 2 0 0 0 2-2V9.026a2 2 0 1 0-4 0v8.952a2 2 0 0 0 2 2Zm-13-3.99c0 7.18 5.82 13 13 13s13-5.82 13-13-5.82-13-13-13-13 5.82-13 13Zm13 5.007a1.5 1.5 0 1 1 0 3 1.5 1.5 0 0 1 0-3Zm1-3.017a1 1 0 1 1-2 0V9.026a1 1 0 1 1 2 0v8.952Z" fill="#FF7B26"/></svg>
```
Comment (abhinavrastogi-harness) [inline packages/icons/src/warning-icon.svg:1]: there's still hardcoded hex colors in the path elements

Comment (harshilbits) [reply packages/icons/src/warning-icon.svg:1]: done.

File: packages/icons/src/remove-minus.svg (hunk 3 above)
Comment (harshilbits) [reply packages/icons/src/remove-minus.svg]: done.

File: packages/icons/src/execution-warning.svg (hunk 2 above)
Comment (harshilbits) [reply packages/icons/src/execution-warning.svg:1]: In storybook...it loads correctly

**Review by abhinavrastogi-harness — APPROVED** [review]
please check if `warning-icon` is used in many places... since we are changing the color from orange to `currentColor`, it might look different now if correct color is not provided via css

**Review by deepesh-ui — APPROVED** [review]

**Pull request merged**